bubble.rb

```diff
@@ -0,0 +1,37 @@
+def bubble_sort(myarray)
+  n = myarray.length
+  loop do
+    swapped = false
+    (n - 1).times do |i|
+      if myarray[i] > myarray[i + 1]
+        myarray[i], myarray[i + 1] = myarray[i + 1], myarray[i]
+        swapped = true
+      end
+    end
+    break if swapped
+  end
+  myarray
+end
+a = [1, 4, 6, 2, 1, 9, 8, 3]
+p bubble_sort(a)
```

Comment:
- This method returns [3, 4, 2, 0, 2, 78] instead of [0, 2, 2, 3, 4, 78] when [4,3,78,2,0,2] is passed.